documentation/MIGRATION guide for bumping up to v30.0.0.md

```diff
@@ -0,0 +1,196 @@
+1. Callbacks: Method overloads that used callbacks have been removed and the use of promises is encouraged instead
+1. List operations now return an iterable result that follows the `PagedAsyncIterableIterator` interface as opposed to the previous model where you had to make a new request using the link to the next page.
+1. Long running operations i.e. the Lro related object returned by methods whose names started with `begin`, now uses `pollUntilDone` to check whether the request is finished, instead of `pollUntilFinished`. To get the final result, use the corresponding method that will have the suffix `AndWait`.
+1. The SDK only supports ECMAScript 2015 (ES6) and beyond, all projects that referenced this SDK should be upgraded to use ES6.
+
+If you have an existing application that uses the Javascript/Typescript Azure SDK packages and you're interested in updating your application to use the newer JavaScript/Typescript Azure SDK packages, then the good news is that there is very little for you to do. Here's the things that have changed with this new set of SDKs:
+
+## Current status
+Currently, we have previewed several packages to version 30.0.0+. See the below list and have a try.  
+1. [@azure/arm-resources](https://www.npmjs.com/package/@azure/arm-resources/v/30.0.0-beta.1)
+1. [@azure/arm-features](https://www.npmjs.com/package/@azure/arm-features/v/30.0.0-beta.2)
+1. [@azure/arm-locks](https://www.npmjs.com/package/@azure/arm-locks/v/30.0.0-beta.1)
+1. [@azure/arm-links](https://www.npmjs.com/package/@azure/arm-links/v/30.0.0-beta.1)
+1. [@azure/arm-policy](https://www.npmjs.com/package/@azure/arm-policy/v/30.0.0-beta.1)
+1. [@azure/arm-managedapplications](https://www.npmjs.com/package/@azure/arm-managedapplications/v/30.0.0-beta.1)
+
+## Authentication
+
+In version 30.0.0+ Javescript/Typescript packages, we only `@azure/identity` to do the Authentication. And we have deprecated the authentication method of `@azure/ms-rest-nodeauth` and `@azure/ms-rest-browserauth`. If you are using them, you may change it accordingly.  
+For example, if you are using `loginWithServicePrincipalSecret` method in `@azure/ms-rest-nodeauth` to get the credential, you may replace it with `ClientSecretCredential` in `@azure/identity`.  
+
+change
+```typescript
+import * as msRestNodeAuth  from '@azure/ms-rest-nodeauth';
+const credentials = msRestNodeAuth.loginWithServicePrincipalSecret(clientId, clientSecret, tenantId);
+```
+into
+```typescript
+import { ClientSecretCredential }  from '@azure/identity';
+const credentials = new ClientSecretCredential(tenantId, clientId, clientSecrat);
+```
+Refer to [@azure/identity](https://www.npmjs.com/package/@azure/identity) for more details.
+
+
+## Callbacks
+
+In version below 30.0.0. we have some operations that allow users to use callback like :
+
```

Comment:
better to use a heading above such as "Old" , "New" to make it clear

Reply:
Added current libraries and next generation as two title.